lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -202,7 +203,7 @@ class window.Turbolinks
       triggerEvent 'page:change'
       triggerEvent 'page:update'
 
-    return
+    return [].slice.call(body.children)
```

Comment:
I think it can't really hurt
I'd be surprised if anyone was expecting `undefined` or `null` return value here
I guess one could argue it's doing extra computation